package.json

```diff
@@ -23,31 +23,31 @@
     "swr": "^2.2.5"
   },
   "devDependencies": {
-    "@commitlint/cli": "19.1.0",
+    "@commitlint/cli": "19.2.1",
```

Comment:
does upgrading dependencies have any improvements in the website performance?

otherwise please revert these changes and keep the pr as simple as possible

Reply:
Is good to have them updated, but I can revert it.